vst.go

```diff
@@ -745,11 +748,13 @@ const (
 type (
 	// Parameter refers to plugin parameter that can be mutated in the pipe.
 	Parameter struct {
-		Name         string
-		Unit         string
-		Value        float32
-		ValueLabel   string
-		NotAutomated bool
+		Name            string
+		Unit            string
+		Value           float32
+		ValueLabel      string
+		NotAutomated    bool
+		GetParamDisplay func(Value float32) string
+		GetValue        func(Value float32) float32
```

Comment:
1. rename to `GetParamDisplayFunc` and `GetValueFunc`
2. rename `Value` parameter to `value`

Reply:
I've changed the naming.